pkg/resourceapply/rbac_test.go

```diff
@@ -1112,25 +1115,28 @@ func TestApplyRoleBinding(t *testing.T) {
 			expectedEvents:  nil,
 		},
 		{
-			name: "updates the RB when AutomountRoleBindingToken differ",
+			name: "recreates the RB when roleRef changes",
```

Comment:
test with update of AutomountRoleBindingToken could stay, new one for recreate could be added